index.js

```diff
@@ -18,12 +18,18 @@ function findAllAfter(parent, index, test) {
   children = parent.children
   length = children.length
 
-  if (index && index.type) {
+  if (index === undefined) {
```

Comment:
```suggestion
  if (index === undefined || index === null) {
```

`null` and `undefined` are typically treated the same, so I think for them to have the same error message here makes sense.